doc/user_guide/user_guide.md

```diff
@@ -1,5 +1,111 @@
 # Exasol AWS Glue Connector User Guide
 
+## Table of Contents
+
+- [Preparing Required Services](#preparing-required-services)
+- [Setting up the Connector](#setting-up-the-connector)
+- [Job Parameters](#job-parameters)
+- [Data Mapping](#data-mapping)
+
+## Preparing Required Services
+
+To use the AWS Glue Studio connector, you should prepare the following steps which then will be used setting up the connector.
+
+### Creating IAM Role
+
+We are going to create an IAM Policy Role with required permissions that will act on our behalf when using AWS Glue Studio Connector. Go to ["IAM Service"](https://console.aws.amazon.com/iam/) &rarr; from navigation page choose "Roles" &rarr; click "Create Role" on the top right corner. Please follow these steps:
```

Comment:
As an alternative we could also show instructions using the CLI. Then we could specify the roles & permissions directly.

Reply:
Sure, it would be nice. I created separate issue #48 for it.